tests/fixtures/debian/kafka-streams-examples/standalone-config.yml

```diff
@@ -0,0 +1,51 @@
+---
+version: '2'
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:3.2.0
```

Comment:
Hmm, I see.  But there is not `:3.2.x` for "latest" of the 3.2.x release line, right?  I think I understand why this would work for `tests`, because it is (a) based on a *local* build AND (b) because it deletes any existing `confluentinc/*` images prior to the build (hence `:latest` should always be "the latest for this particular current branch").  Is that how it is?